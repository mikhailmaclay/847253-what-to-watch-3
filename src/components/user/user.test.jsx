// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import User from './user';

describe(`<User/>`, () => {
  const props = {
    name: `Shia LaBeouf`,
    avatar: ``
  };

  it(`Should render correctly`, () => {
    const result = renderer.create(
        <User {...props}/>
    ).toJSON();

    expect(result).toMatchSnapshot();
  });
});
