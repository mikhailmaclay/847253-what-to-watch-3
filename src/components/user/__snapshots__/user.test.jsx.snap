// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User/> Should render correctly 1`] = `
<div
  className="user-block"
>
  <a
    className="user-block__link"
    href="/sign-in"
    onClick={[Function]}
  >
    Sign in
  </a>
</div>
`;
